variables-workers.tf

```diff
@@ -147,6 +147,12 @@ variable "worker_shape" {
   type        = map(any)
 }
 
+variable "worker_capacity_reservation_id" {
+  default     = null
+  description = "The ID of the Compute capacity reservation the worker node will be launched under"
```

Comment:
Can we add a reference to docs here, e.g.
> The ID of the Compute capacity reservation the worker node will be launched under. Refer to <a href=https://docs.oracle.com/en-us/iaas/Content/Compute/Tasks/reserve-capacity.htm>Capacity Reservations</a> for more information.

Links will be rendered with documented inputs.